accessibility/common-remediations.md

```diff
@@ -416,6 +419,85 @@ We _don’t_ do this:
 <title>Checkout | My webpage</title>
 ```
 
+## Language of page and parts
+
+### Specified language is incorrect
+
+#### What's the problem?
+
+The language specified in the HTML `lang` attribute doesn't match the language used for the main page. 
+
+#### Who's affected by the problem?
+
+* Screen reader users
+
+#### Why's it a problem?
+
+Words are pronounced differently in different languages. Screen reader software is designed to respect these pronouncation differences when it encounters web pages that specify their main language. 
```

Comment:
pronouncation?

Reply:
oops, yes. I always misspell that

```suggestion
Words are pronounced differently in different languages. Screen reader software is designed to respect these pronunciation differences when it encounters web pages that specify their main language. 
```

Reply:
oh i merged already. will put it in another PR